src/Symfony/Component/Intl/Tests/DateFormatter/AbstractIntlDateFormatterTest.php

```diff
@@ -355,13 +351,9 @@ public function testFormatWithConstructorTimezone()
 
     public function testFormatWithDateTimeZone()
     {
-        if (defined('HHVM_VERSION_ID')) {
-            $this->markTestSkipped('This test cannot work on HHVM. See https://github.com/facebook/hhvm/issues/5875 for the issue.');
-        }
-
-        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT+03:00'), IntlDateFormatter::GREGORIAN, 'zzzz');
+        $formatter = $this->getDateFormatter('en', IntlDateFormatter::MEDIUM, IntlDateFormatter::SHORT, new \DateTimeZone('GMT'), IntlDateFormatter::GREGORIAN, 'zzzz');
```

Comment:
well, the goal was to ensure that this syntax was supported when converting DateTimeZone to Intl timezone ids (there is a specific handling for them).

We might add another test using GMT only and keep the test with ``GMT+03:00`` but skipping it in versions not supporting it.

Btw, this should be done in 2.3+ if we go this way